src/infrastructure/figma/transformers/figma-file-meta-transformer.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	buildDesignUrl,
+	buildInspectUrl,
+	buildLiveEmbedUrl,
+	getUpdateSequenceNumberFrom,
+} from './utils';
+
+import type { AtlassianDesign } from '../../../domain/entities';
+import {
+	AtlassianDesignStatus,
+	AtlassianDesignType,
+	FigmaDesignIdentifier,
+} from '../../../domain/entities';
+import type { GetFileMetaResponse } from '../figma-client';
+
+type TransformFileMetaToAtlassianDesignParams = {
+	readonly fileKey: string;
+	readonly fileMetaResponse: GetFileMetaResponse;
+};
+
+/**
+ * Transforms a Figma file medata to {@link AtlassianDesign}.
+ */
+export const transformFileMetaToAtlassianDesign = ({
+	fileKey,
+	fileMetaResponse,
+}: TransformFileMetaToAtlassianDesignParams): AtlassianDesign => {
+	const designId = new FigmaDesignIdentifier(fileKey);
+	const fileName = fileMetaResponse.file.name;
+
+	return {
+		id: designId.toAtlassianDesignId(),
+		displayName: fileMetaResponse.file.name,
+		url: buildDesignUrl({ fileKey, fileName }),
+		liveEmbedUrl: buildLiveEmbedUrl({ fileKey, fileName }),
+		inspectUrl: buildInspectUrl({ fileKey, fileName }),
+		status: AtlassianDesignStatus.NONE,
```

Comment:
Just confirming this is always NONE because status is only available on sections?

Reply:
This is correct, Sam. 

`status` is available only for Figma Nodes:
https://github.com/atlassian-labs/figma-for-jira/blob/e6e7c80efcf519e01bc8d5b3d48d0a4f459776d8/src/infrastructure/figma/transformers/figma-node-transformer.ts#L37